index.html

```diff
@@ -79,22 +85,43 @@ <h2 class="backend">Back End Dev</h2>
                         <h2 class="year">2015</h2>
                     </li>
                 </ul>
-            </div>
-            <p class="p-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="framed">
+                    <li>
+                        <h2 class="canopy">CANOPY</h2>
+                    </li>
+                    <li>
+                        <img src="Counter.png" alt="">
+                    </li>
+                    <li>
+                        <h2 class="backend">Back End Dev</h2>
+                    </li>
+                    <li>
+                        <img src="Counter.png" alt="">
+                    </li>
+                    <li>
+                        <h2 class="year">2015</h2>
+                    </li>
+                </ul>
+                <p class="p-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <p class="p-textd">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
             <ul class="lang">
                 <li class="p-html"><a class="a-lang" href="">html</a></li>
                 <li class="p-css"><a class="a-lang" href="">css</a></li>
                 <li class="p-javas"><a class="a-lang" href="">javaScript</a></li>
             </ul>
-            <div class="p-button">
-                <a class="p-proj" href="">See Project</a>
+            <button class="p-button" type="submit">
+                See Project
+            </button>
```

Comment:
- [ ] [OPTIONAL] For all you button, you can consider to add the next CSS styles to improve the UX in your website.

```
button:hover {
  cursor: pointer;
}
```